devel/spdlog/Portfile

```diff
@@ -51,24 +53,24 @@ variant fmt_external description {Use external fmt library instead of bundled} {
     patchfiles-append \
                     patch-spdlog-fmt-external.diff
     cmake.module_path-append \
-                    ${prefix}/lib/libfmt10/cmake
+                    ${prefix}/lib/libfmt${libfmt_ver}/cmake
     depends_lib-append \
-                    port:libfmt10
+                    port:libfmt${libfmt_ver}
     configure.args-replace \
                     -DSPDLOG_FMT_EXTERNAL=OFF \
                     -DSPDLOG_FMT_EXTERNAL=ON
 }
 
-variant tests description {Enable test support} {
```

Comment:
Just curious, did you intend to drop variant `tests`?

That's definitely useful, and it should remain.

Reply:
Oops. It wasn't intentional. Will revert soon.

Reply:
> Oops. It wasn't intentional. Will revert soon.

No worries, glad it wasn't intended!